test/AutoDiff/stdlib/callee_differential_not_leaked_in_func_with_loops.swift

```diff
@@ -0,0 +1,61 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import _Differentiation
+import StdlibUnittest
+import CoreFoundation
```

Comment:
I do not think this will work on Linux / Windows – no CoreFoundation there. Can we use the existing `Tracked` wrappers btw?

Reply:
Ah sorry, that CoreFoundation import is unused from. I was trying something out. I'll remove it.